Annotated Dockerfile (#20)
laminas/laminas-mvc-skeleton: README.md

This skeleton provides a `docker-compose.yml` for use with
[docker-compose](https://docs.docker.com/compose/); it

> uses the `Dockerfile` for a `laminas` container.  Build and start the image 
> using:

Reviewer: I think it would be more clear if worded something like this:
```suggestion
uses provided `Dockerfile` to build docker image and run docker-compose service named `laminas`.
Build and start the image using:
```

Author: Applied and edited

```bash
$ docker-compose up -d --build
```

At this point, you can visit http://localhost:8080 to see the site running.

> You can also run commands such as composer from the image.  The container 

Reviewer: ```suggestion
You can also run commands such as `composer` in the container.  The container 
```

environment is named "laminas" so you will pass that value to 
`docker-compose run`:

```bash
$ docker-compose run laminas composer install
```

> Some optional modules use the `intl` PHP extension so the Dockerfile contains
> commented commands to enable this extension and others.  To install `intl` 
> support uncomment the lines

Reviewer: Make it less specific so that making intl pre-installed and redis optional would need fewer changes.
```suggestion
Some composer packages need additional PHP extensions, the Dockerfile contains several
examples of commented out commands that add some of the more popular php extensions.
For example, to install `intl` support uncomment the lines
```

Also, in the suggestion I removed mention of optional modules as I believe all the optional packages that can be installed via skeleton installer should have all the php-* dependencies already available in the image.

Author: Applied and edited.
